xwiki-platform-core/xwiki-platform-xclass/xwiki-platform-xclass-ui/src/main/resources/XWiki/ObjectSheet.xml

```diff
@@ -42,7 +42,7 @@
 
 #set($class = $doc.getObject('XWiki.MyClass').xWikiClass)
 #foreach($prop in $class.properties)
-  ; $prop.prettyName
+  ; $doc.displayPrettyName($prop.name)
   : $doc.display($prop.getName())
```

Comment:
Looks good. You can also use the short form of the `$prop.getName()` in the second line, for consistency.